CONTRIBUTING.md

```diff
@@ -86,10 +86,23 @@ snapshot builds of the `master` branch. They are available from
 
 Build using Java 11:
 
-```gradle assemble```
+```bash
+java -version
+```
+
+```bash
+./gradlew assemble
+```
+
+and then generate the -all artifact
+
+```bash
+./gradlew :opentelemetry-javaagent:shadowJar
+```
```

Comment:
Good catch! I opened #708 to see about generating the `-all` artifact as part of `assemble` in the future, but good to update docs now to reflect the current state.